### DISCUSSION TITLE
where to add cordova variable in capacitor

### DISCUSSION THREAD

**Discussion by indraraj26** [ionic-team/capacitor#3522] [Q&A] [answered]
I would like to add google plus 
```
<plugin name="cordova-plugin-googleplus" source="npm">
  <variable name="REVERSED_CLIENT_ID" value="myreversedclientid" />
  <variable name="WEB_APPLICATION_CLIENT_ID" value="mywebapplicationclientid" />
</plugin>
```

**Comment by imhoffd** [comment, marked as answer]
Please see [this section](https://capacitorjs.com/docs/cordova/using-cordova-plugins#important-configuration) in the docs. Cordova variables aren't fully supported in Capacitor. You'll have to investigate how each plugin uses each variable and apply those changes to your project.

For example:
- `REVERSED_CLIENT_ID`: looks like you'll have to edit `Info.plist` (https://github.com/EddyVerbruggen/cordova-plugin-googleplus/blob/8fd99c948a7e46c41855b855221438fdfb3fe9bc/plugin.xml#L69-L82)
- `WEB_APPLICATION_CLIENT_ID`: used during `after_prepare` hook for web to write the client ID into a source file (https://github.com/EddyVerbruggen/cordova-plugin-googleplus/blob/82d12c6b9a77f0395621f0419ef241ead756eaa2/hooks/browser/after_prepare.js)

**Reply by indraraj26** [reply]
for now its look like i should rollback to cordova, It will break so many things of cordova, It is really hard to understand where to put those values

Thank you
